scripts/build_schemas.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-mkdir -p data/en
+rm -rf data/en && mkdir -p data/en
```

Comment:
We need to be careful about this as people could have things they're working on in that directory. Maybe we should have a `make clean` command?

Reply:
They should be working within `data-source/json/` and not `data/en`?

Reply:
Most of the time yes, but I've made changes to schemas in data/en before to try things out without rebuilding.

Reply:
Is that a pain point of not wanting to wait for all the census schemas to build?. I do that too. 
For now, I have introduced a new `clean` command to delete the `data` dir and a `build-schemas` command to build just the schemas not everything such as translations.